test/pom.xml

```diff
@@ -53,7 +53,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.36</version>
+      <version>2.41-20180911.205237-1</version>
```

Comment:
please add a `TODO` comment linking to upstream

Reply:
Anyway I released 2.42.

Reply:
#3661 picks it up